add parameter for station core and habitat

LouisBarboutie opened the pull request:
It adds the parameters to have a core of a certain size and a habitat in the same size ballpark.

LouisBarboutie committed:
add parameter for station core and habitat

linuxgurugamer commented:
I saw that you opened and then closed a PR, is this complete yet?

LouisBarboutie commented:
Yes. I only modified the sspx.cfg-disabled file though, but I can touch up the combined one next if needed.
The close was accidental, I am not very used to github, and didn't know how to update my fork. I ended up deleting my whole fork to renew it. The PR must've been closed by that action.

linuxgurugamer commented:
I don't think you understand what was done here.  All those "disabled" files are just that, disabled, they were replaced by the StationCoreCombined file.

LouisBarboutie commented:
Aah, ok. That was quite confusing how it was organized. It should be straightforward to implement it into the combined config though. I'll get onto it soon.

LouisBarboutie committed:
change the correct file

LouisBarboutie committed:
Streamlined Parameters

Reworked needing having the same size hab and core. Now only checks for any core and any hab.

linuxgurugamer commented:
You renamed the file, I'll fix that before releasing it
